tooling/nargo_cli/src/cli/test_cmd.rs

```diff
@@ -42,8 +48,28 @@ pub(crate) struct TestCommand {
     /// JSON RPC url to solve oracle calls
     #[clap(long)]
     oracle_resolver: Option<String>,
+
+    /// Number of threads used for running tests in parallel
+    #[clap(long)]
+    test_threads: Option<usize>,
```

Comment:
If you want you can provide the default value right here, see an [example](https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=731f0008d73402fd93f44c54b5c7e2f0)